src/module/migration/migrations/779-elite-weak.ts

```diff
@@ -0,0 +1,16 @@
+import { ActorSourcePF2e } from "@actor/data";
+import { MigrationBase } from "../base";
+
+export class Migration779EliteWeak extends MigrationBase {
```

Comment:
```suggestion
/** Store indication of elite/weak adjustment on boolean property instead of as trait */
export class Migration779EliteWeak extends MigrationBase {
```